docs/request.rst

```diff
@@ -89,6 +89,34 @@ supports collection formats "pipes" and "csv". The default format is "csv".
 
 .. _jsonschema: https://pypi.python.org/pypi/jsonschema
 
+Nullable parameters
+^^^^^^^^^^^^^^^^^^^
+
+Sometimes your API should explicitly accept _`nullable parameters`. However
+OpenAPI specification currently does _`not support` officially a way to serve
+this use case, Connexion adds the `x-nullable` vendor extension to parameter
+definitions. It's usage would be:
```

Comment:
I would add a note saying than if/when a future version of OpenAPI defines an option for this, we will deprecate this extension.